### PR TITLE
Add Konflux build file

### PR DESCRIPTION
**Pull request by dhaiducek**
**Related Issue:**  https://issues.redhat.com/browse/ACM-10719

**Description of Changes:** Adds Dockerfile for Konflux builds

**What resource is being added**: N/A

**Is this a Hub or Managed cluster change?:** N/A

**Notes:** Also contains minor repo cleanup

### CODE REVIEW

**Review by Kyl-Bempah — APPROVED** [review]
LGTM

**Review by Kyl-Bempah — APPROVED** [review]
/lgtm
/approve

File: Dockerfile.rhtap [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+FROM registry.redhat.io/openshift4/ose-cli:v4.15 as builder
+
+FROM registry.access.redhat.com/ubi9/ubi-minimal:latest
+
+RUN microdnf update -y \
+    && microdnf install -y tar rsync findutils gzip iproute util-linux wget \
+    && microdnf clean all
+
+RUN wget https://github.com/mikefarah/yq/releases/download/v4.2.0/yq_linux_amd64 -O /usr/bin/yq &&\
```
Comment (jlpadilla) [inline Dockerfile.rhtap:12]: Should the yq version match the Dockerfile (v4.44.2) ?

Comment (dhaiducek) [reply Dockerfile.rhtap:12]: Oops--good catch! Apparently I updated it after copying it. 🤦‍♂️

**Review by jlpadilla — APPROVED** [review]
/lgtm

**Pull request merged**